src/devices/Device-Index.md

```diff
@@ -35,6 +35,7 @@
 * [Mcp23xxx - I/O Expander device family](Mcp23xxx/README.md)
 * [Mcp25xxx device family - CAN bus](Mcp25xxx/README.md)
 * [Mcp3428 - Analog to Digital Converter (I2C)](Mcp3428/README.md)
+* [Mbi5027 -- 16-bit shift register](Mbi5027/README.md)
```

Comment:
Device-Index .md is generated automatically. What you need is the category.txt file in the same directory as your csproj